BUILDING.md

```diff
@@ -61,19 +61,22 @@ note1 - The gcc4.8-libs package needs to be installed, because node
   by Joyent. SmartOS images >= 16.4 are not supported because
   GCC 4.8 runtime libraries are not available in their pkgsrc repository
 
-*Note*: On Windows, running Node.js in windows terminal emulators like `mintty`
-  requires the usage of [winpty](https://github.com/rprichard/winpty) for
-  Node's tty channels to work correctly (e.g. `winpty node.exe script.js`).
+<em id="fn2">2</em>: Teir 1 support for building on Windows is only on 64 bit
+  hosts. Support is experimental for 32 bit hosts.
+
+<em id="fn3">3</em>: On Windows, running Node.js in windows terminal emulators
+  like `mintty` requires the usage of [winpty](https://github.com/rprichard/winpty)
+  for Node's tty channels to work correctly (e.g. `winpty node.exe script.js`).
```

Comment:
`Node's` -> `Node.js's`? (A bit awkward, but it seems we try to be consistent here.)

Reply:
How about `Node's` -> `the`

Reply:
Seems good.